Client/qtTeamTalk/settings.h

```diff
@@ -28,9 +28,9 @@
 #include <QApplication>
 
 #if defined(Q_OS_WIN32)
-#define SOUNDSPATH                                     QApplication::applicationDirPath()+"/Sounds"
+#define SOUNDSPATH                                     "Sounds"
```

Comment:
Doesn't this cause the use of a relative path instead of an absolute path?

The ChangeLog.txt status: "- Use absolute file path when changing sound pack"

Reply:
Yes, you're right, I mistaken in change log. Fixed.